compiler/fbs/Converter.hx

```diff
@@ -71,18 +68,17 @@ class Converter {
 		});
 
 		var printer:haxe.macro.Printer = new haxe.macro.Printer();
-		currentModule.types.map(function(t) {
+        for (t in currentModule.types){
```

Comment:
The Lambda.map functions changed (to require a return).  This expression works fine as a for-loop.